src/spec.c

```diff
@@ -2544,7 +2544,7 @@ static void Indexes_LoadingEvent(RedisModuleCtx *ctx, RedisModuleEvent eid, uint
     if (hasLegacyIndexes || CompareVestions(redisVersion, noScanVersion) < 0) {
       Indexes_ScanAndReindex();
     } else {
-      RedisModule_Log(ctx, "warning",
+      RedisModule_Log(ctx, "notice",
```

Comment:
Notice will still show the warning by default (default log level is notice), maybe we want to change it to verbose?

Reply:
We can.
I changed to `notice` because that’s the level we use in other logs on loading events, also inside the `Indexes_ScanAndReindex` call.
But since it should be the default behavior we can reduce it to `verbose`